community/kernel/src/main/java/org/neo4j/kernel/impl/api/OperationsFacade.java

```diff
@@ -1541,6 +1541,7 @@ public void unregisterExecutingQuery( ExecutingQuery executingQuery )
             BasicContext ctx = new BasicContext();
             ctx.put( Context.KERNEL_TRANSACTION, tx );
```

Comment:
Since ProcedureGDBFacadeSPI has a currentTransaction method, do we need the kernel transaction in the context also?